apps/cli/src/apis.ts

```diff
@@ -15,7 +15,7 @@ export const validateAccessToken = async (
 	accessToken: string,
 ): Promise<Data> => {
 	const response = await Axios.post(
-		"https://backend.codemod.com/validateAccessToken",
+		"http://localhost:8081/validateAccessToken",
```

Comment:
should we move base url to env?

Reply:
good idea, we don't actually have env currently in CLI, but we can taskify this. i wonder how will it be packaged to the `index.cjs` binary.